nixpkgs_review/report.py

```diff
@@ -117,12 +138,34 @@ def write(self, directory: Path, pr: Optional[int]) -> None:
         with open(directory.joinpath("report.json"), "w+") as f:
             f.write(self.json(pr))
 
-        write_error_logs(self.attrs, directory)
+        write_result_links(self.attrs, directory)
+        write_error_logs(self.failed, directory)
 
     def succeeded(self) -> bool:
         """Whether the report is considered a success or a failure"""
         return len(self.failed) == 0
 
+    def upload_build_logs(self, github_client: GithubClient, pr: Optional[int]) -> None:
```

Comment:
We could also implement the fancy logs with colors here https://github.com/SuperSandro2000/nixpkgs-review-checks/blob/d07c4164b14a4fe989acb25eb1428dd795384e26/nixpkgs-review-checks#L323.

Reply:
I think in this case, it makes sense to put this feature (uploading the logs) into `nixpkgs-review` rather than having it as a custom extension. Doing it as a `gist` allows us to re-use the same GITHUB_TOKEN infrastructure, which makes a lot of sense.

Reply:
I wanted to say that we can also throw the colored logs into nixpkgs-review

Reply:
Maybe we can do that in a second PR?